ambry-router/src/main/java/com.github.ambry.router/SimpleOperationTracker.java

```diff
@@ -66,27 +69,32 @@
 class SimpleOperationTracker implements OperationTracker {
   protected final String datacenterName;
   protected final String originatingDcName;
-  protected final int successTarget;
-  protected final int parallelism;
+  protected final int diskSuccessTarget;
+  protected final int diskParallelism;
+  protected final int cloudSuccessTarget;
+  protected final int cloudParallelism;
+  protected final boolean cloudReplicasPresent;
+  // How many NotFound responses from originating dc will terminate the operation.
+  // It's decided by the success target of each mutation operations, including put, delete, update ttl etc.
+  protected final int originatingDcNotFoundFailureThreshold;
```

Comment:
Sorry, I am not sure if I wrote the comments, but this can be applied to GET as well

Reply:
Changed the comment. The success target no longer has a part in this calculation after the dynamic target changes